tests/benchmarks/multicore/idle_spim/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&spi131 {
+	zephyr,pm-device-runtime-auto;
```

Comment:
Shouldn't this be set at the board definition?
At least @gmarull is doing that for CAN in https://github.com/zephyrproject-rtos/zephyr/pull/78854/files